src/hooks/v3/useUsersV3POOLTokenBalances.ts

```diff
@@ -32,7 +32,7 @@ export const useUsersV3POOLTokenBalances = (usersAddress: string) => {
 
     queriesResult.forEach((queryResult) => {
       const { data: balancesForChainId } = queryResult
-      balancesForChainId.balances.forEach((balance) => {
+      balancesForChainId?.balances.forEach((balance) => {
```

Comment:
`isFetched` sometimes returns true when the data still isn't there. I believe it's typically when there is an `error`, so `isFetched` is true and `error` is true and `data` is `null`. We try to do something with that data but the app crashes because it wasn't there.